assets/js/index.js

```diff
@@ -79,15 +79,30 @@ $("#copybutton").click(function () {
 
   const message = dropdownMap.get(id);
   let messageBody = getStringValue(message.message);
+
   // Replace predefined variables in the message body
-  for (const variable in variables) {
-    for (const { project: expected, value } of variables[variable]) {
-      if (isExpectedProject(expected, project)) {
-        messageBody = messageBody.replace(new RegExp(`%${variable}%`, 'g'), getStringValue(value));
+  function resolveVariables(text, limit) {
+    var varExpanded = false;
+    for (var variable in variables) {
+      if (text.includes(variable)) {
+        for (const { project: expected, value } of variables[variable]) {
+          if (isExpectedProject(expected, project)) {
+            varExpanded = true;
+            text = text.replace(new RegExp(`%${variable}%`, 'g'), getStringValue(value));
+          }
+        }
       }
     }
+
+    if (varExpanded && limit > 0) {
+      return resolveVariables(text, limit - 1);
+    } else {
+      return text;
+    }
   }
-  // TODO: Allow more than one variable
```

Comment:
(just a small note: the comment here is actually for the `message.fillname` part)

Reply:
Ah, whoops, i thought it was about what I implemented.
Anyway, if that's really needed, it should probably be an issue at this point, but I don't really see the need for multiple inputs as of right now.